webapp/channels/src/components/delete_category_modal/delete_category_modal.tsx

```diff
@@ -72,3 +74,11 @@ export default class DeleteCategoryModal extends React.PureComponent<Props, Stat
         );
     }
 }
+
+// TODO MM-51399 These strings are properly defined in @mattermost/components, but the i18n tooling currently can't
+// find them there, so we've had to redefine them here
```

Comment:
Also, it seems this was done before at the `GenericModal` level, why are we moving this to this specific modal?

Reply:
> [mattermost.atlassian.net/browse/MM-51399](https://mattermost.atlassian.net/browse/MM-51399) refers to the whole epic of post monorepo work. Is there a specific ticket you are referring here that needs to be fixed?

Added the correct ticket

> Also, it seems this was done before at the GenericModal level, why are we moving this to this specific modal?

Yes as mentioned in description that file is now removed, so until the translations of platform folder is sorted this can live here